executor/nodes_stat.go

```diff
@@ -0,0 +1,162 @@
+package executor
+
+import (
+	"admin-cli/helper"
+	"context"
+	"fmt"
+	"github.com/XiaoMi/pegasus-go-client/idl/admin"
+	"github.com/olekukonko/tablewriter"
+	"strconv"
+	"time"
+)
+
+type NodeStat struct {
+	Node      string
+	Status    string
+	GetQPS    int64
+	MgetQPS   int64
+	PutQPS    int64
+	MputQPS   int64
+	GetBytes  int64
+	MgetBytes int64
+	PutBytes  int64
+	MputBytes int64
+	// nodes-stat -d
+	DiskTotalMB  int64
+	DiskUsedMb   int64
+	DiskRatio    int64
+	MemUsedMB    int64
+	BlockCacheMB int64
+	MemTableMB   int64
+	MemIdxMB     int64
+}
+
+func NodesStat(client *Client, detail bool) error {
```

Comment:
```suggestion
func ShowNodesStat(client *Client, detail bool) error {
```

Do not give the same to both struct and function. This rule is applicable to every programming language.

Reply:
Right, the struct is `nodeStat`, but function is `nodesStat`